src/test/java/streams/part1/exercise/Exercise1.java

```diff
@@ -19,23 +18,29 @@
     public void findPersonsEverWorkedInEpam() {
         List<Employee> employees = Example1.getEmployees();
 
-        // TODO реализация
-        List<Person> personsEverWorkedInEpam = null;
+        List<Person> personsEverWorkedInEpam = employees.stream()
+                .filter(employee -> employee.getJobHistory().stream()
+                        .anyMatch(job -> job.getEmployer().equals("EPAM")))
```

Comment:
As alternative we can use:
`.map(JobHistoryEntry::getEmployer)`
`.anyMatch("EPAM"::equals)`